### PR TITLE
Multi-Z ZAS in new connection system hotfix

### PR DESCRIPTION
**Pull request by ghost**
[bugfix][hotfix][system]
:cl:
 * bugfix: Multi-Z ZAS now connects properly to any non adjacent z-level linked above or below, as to fit into the new connection system.

### CODE REVIEW

File: code/ZAS/Turf.dm [hunk 1]
```diff
@@ -72,12 +72,21 @@
...
 	#ifdef ZLEVELS
-	for(var/d = 1, d < 64, d *= 2)
+	for(var/d = 1, d <= 32, d *= 2)
 	#else
-	for(var/d = 1, d < 16, d *= 2)
+	for(var/d = 1, d <= 8, d *= 2)
 	#endif
 
+		#ifdef ZLEVELS
+		if(d = 16)
```
Comment (Exxion) [inline code/ZAS/Turf.dm]: `=`

File: code/ZAS/Turf.dm [hunk 2]
```diff
@@ -72,12 +72,21 @@
...
+			var/turf/unsim = GetAbove(src)
+		else if(d = 32)
```
Comment (Exxion) [inline code/ZAS/Turf.dm]: `=`

File: code/ZAS/Turf.dm (hunk 1 above)
Comment (Exxion) [reply code/ZAS/Turf.dm]: Oh also you should use the defines

File: code/ZAS/Turf.dm (hunk 2 above)
Comment (ghost) [reply code/ZAS/Turf.dm]: Fixed

File: code/ZAS/Turf.dm (hunk 1 above)
Comment (ghost) [reply code/ZAS/Turf.dm]: Fixed, added defines too

**Review by Exxion — APPROVED** [review]

**Pull request merged**